### PR TITLE
Add Document to Person class

### LINKED ISSUE #91: [v1.2.1] Add Document Class and Reference Class

### PR DESCRIPTION
**Pull request by CodyChew**
-Document Contructor takes in a String name and String reference.
-Stores it as a single String value "name, reference" in the Json file
-Has no validation for reference and only checks for empty name
-Include minor test cases for empty name and empty document

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Document.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import static java.util.Objects.requireNonNull;
+
+public class Document {
+
+    public static final String MESSAGE_CONSTRAINTS = "Document names should be alphanumeric";
+    public static final String NAME_VALIDATION_REGEX = "\\p{Alnum}+";
+
+    public final String name;
+    public final String reference;
```
Comment (Vielheim) [inline src/main/java/seedu/address/model/person/Document.java]: Would be good to OOP this.

**Review by Vielheim — CHANGES_REQUESTED** [review]

**Review by Vielheim — APPROVED** [review]
